src/coreclr/src/jit/importer.cpp

```diff
@@ -964,16 +964,6 @@ GenTreeCall::Use* Compiler::impPopCallArgs(unsigned count, CORINFO_SIG_INFO* sig
             if (corType != CORINFO_TYPE_CLASS && corType != CORINFO_TYPE_BYREF && corType != CORINFO_TYPE_PTR &&
                 corType != CORINFO_TYPE_VAR && (argRealClass = info.compCompHnd->getArgClass(sig, argLst)) != nullptr)
             {
-                // Everett MC++ could generate IL with a mismatched valuetypes. It used to work with Everett JIT,
```

Comment:
@BruceForstall, I could not track when this workaround was introduced (it was before 2010), maybe you know if it is still needed?

Reply:
Bruce told me offline that Everett MC++ was used around 2003.